### PR TITLE
new version of BloqadeGates

### PR DESCRIPTION
**Pull request by ChenZhao44**

### CODE REVIEW

File: lib/BloqadeGates/Project.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 name = "BloqadeGates"
 uuid = "9b473568-af80-491c-8f51-e251334462c1"
-version = "0.1.0"
+version = "0.1.1"
```
Comment (Roger-luo) [inline lib/BloqadeGates/Project.toml:3]: why change the version number without any updates in the code?

Comment (ChenZhao44) [reply lib/BloqadeGates/Project.toml:3]: The codes changed in the last PR #472 but I forgot to update the version number.

**Review by Roger-luo — APPROVED** [review]
LGTM

**Comment by ChenZhao44** [comment]
@Roger-luo Could you register this new version?

**Pull request merged**